src/style.scss

```diff
@@ -10,21 +10,29 @@
 		justify-content: center;
 		overflow: hidden;
 		position: relative;
-		margin: 0px;
+		margin: 0;
+		z-index: 1;
 
-		&:before {
+		&::before {
 			align-items: center;
 			content: attr(data-emoji-flag);
 			display: flex;
 			filter: blur(3vh) saturate(3) contrast(1) opacity(0.4);
 			font-size: calc(800px + 3vh);
 			height: 100%;
 			justify-content: center;
-			left: 0px;
+			left: 0;
 			pointer-events: none;
 			position: absolute;
-			top: 0px;
+			top: 0;
 			width: 100%;
+			z-index: -1;
+		}
+
+		.alignfull &::before,
+		[data-align="full"] &::before {
+			font-size: calc(1.5 * 100vw);
+			z-index: -1;
 		}
```

Comment:
This fixes the visual regressions. I'm not sure, though, if it should be dependant only on the `alignfull` class name. The z-index fix should apply universally, I guess.